website/blog/2023-12-15-ferretdb-neon-add-mongodb-compatibility.md

```diff
@@ -0,0 +1,236 @@
+
+## Advantages of using FerretDB
+
+Let's take a look at some of these benefits of using FerretDB:
+
+### MongoDB compatibility
+
+The best advantage to using FerretDB is the access you get to the syntax, tools, querying language, and commands available in MongoDB, particularly for many common use cases.
+MongoDB is known for its simple and intuitive NoSQL query language which is widely loved by many developers today.
+By using FerretDB, you can enable Postgres-compatible databases like Neon to run MongoDB workloads.
+
+Read more: [MongoDB Compatibility - What's Really Important?](https://blog.ferretdb.io/mongodb-compatibility-whats-really-important/)
+
+### Open source
+
+As an open source document database, you won't be at a risk of vendor lock-in.
+Since MongoDB's license change to Server Side Public License (SSPL), there's [been a lot of disappointment and confusion with regards](https://blog.ferretdb.io/open-source-is-in-danger/) to how it affects many users and what it would mean for their applications.
+According to the Open Source Initiative – the steward of open source and the set of rules that define open source software – SSPL is not considered open source.
+
+In that regard, FerretDB, licensed under Apache 2.0, is a good option for users looking for a MongoDB alternative to return MongoDB workloads back to open source.
+
+### Multiple backend options
+
+At the moment, FerretDB supports Postgres and SQLite backends, with many ongoing works to support other backends.
+Many databases built on Postgres can actually serve as a backend for FerretDB, including Neon.
+That means you can take advantage of all the features available in that backend to scale and manage your database infrastructure without any fear of vendor lock-in.
+
+## Prerequisites
+
+- A Neon project and Postgres URI
+- FerretDB package
+- `mongosh`
+
+## Create a Neon Project
+
+Please refer to the instructions in the [Neon Sign up](https://neon.tech/docs/get-started-with-neon/signing-uphttps://neon.tech/docs/get-started-with-neon/signing-up) docs if this is your first project.
+
+Before getting started with FerretDB, you need to create a Neon project.
+When creating the project, you need to select the version of postgres, and also create a `ferretdb` database.
+
+![Neon project creation](/img/blog/neon-create-project.png)
+
+Once the project is created, Neon will provide a Postgres connection URI for you.
+This URI will serve as the Postgres URI for your FerretDB instance.
+
+## Install FerretDB on Linux
+
+For this project, we will be using the FerretDB `arm` package, and you can access them from the [GitHub release page](https://github.com/FerretDB/FerretDB/releases).
+To do this, we'll need to curl the latest FerretDB package and then install it:
+
+```sh
+curl -LJO https://github.com/FerretDB/FerretDB/releases/download/v1.16.0/ferretdb-linux-arm64.deb
+```
+
+Then install FerretDB with:
+
+```sh
+sudo dpkg -i  ./ferretdb-linux-arm64.deb
+```
```

Comment:
I'd simplify this as

```suggestion
To do this, download and install the latest FerretDB release:

```sh
$ curl -LJO https://github.com/FerretDB/FerretDB/releases/download/v1.16.0/ferretdb-linux-arm64.deb
$ sudo dpkg -i  ./ferretdb-linux-arm64.deb
```
```

The "install" word twice sounds repetitive.
Also, do we have a link where if someone hits it, they are redirected to the latest available version? That'd be neat.

Reply:
Thanks @henvic. @AlekSi Is there a simplified way to get the latest version?

Reply:
The simplest way to use the latest version is to use our Docker image. Is there any particular reason we use a Debian package with the cloud-native serverless database instead?